server/conf/i18n/messages

```diff
@@ -250,6 +250,8 @@ label.inProgress=Not yet submitted
 label.submitted=Submitted
 # Informational tag on a submitted application card. The parameter is the date of the submission.
 label.submittedOn=Submitted on {0}
+# Informational tag on a submitted application card. The first paramater is the status applied to an application. The second parameter is the date the status was applied.
```

Comment:
nit: We usually have it being its own PR when adding new string to be translated.

Reply:
split into https://github.com/civiform/civiform/pull/9398

Reply:
Speaking for myself, I don't usually split it out.